Cargo.toml

```diff
@@ -65,6 +66,9 @@ runtime = [
     "tokio-threadpool",
     "tokio-timer",
 ]
+uds = [
+    "tokio-uds",
```

Comment:
I wouldn't make a separate `uds` feature, just make this part of `runtime`.

Reply:
Well, in that case, it should be actually platform-dependent, since Windows doesn't have Unix Domain Sockets. How does one make a feature depend on another feature (runtime) and also a platform (unix)?